src/test_scenes/sceneSlider.js

```diff
@@ -22,45 +22,42 @@ class SceneSlider extends React.Component {
   render() {
     const { defaultSliderValue, userDefinedSliderOneValue, userDefinedSliderTwoValue } = this.state;
     return (
-      <View position={this.props.position}>
-        <GridLayout
-          anchorPoint={"top-center"}
-          columns={1}
-          defaultItemPadding={[0.075, 0, 0, 0]}
-          position={[0.0, 1.0, 0.0]}
-        >
-          <Text alignment={"center-center"} fontSize={0.075} >
```

Comment:
you're no longer specifying `Alignement`?

Reply:
🙇